subprojects/docs/src/samples/userguide/bestPractices/taskDefinition/build.gradle

```diff
@@ -0,0 +1,18 @@
+import com.enterprise.DocsGenerate
+
+task generateHtmlDocs(type: DocsGenerate) {
+    group = 'Documentation'
```

Comment:
:o: Maybe a stupid suggestion, but I think it'd be a nice touch to use `JavaBasePlugin.DOCUMENTATION_GROUP`, because it hints at use of the base plugin and uses a known standardized group name

Reply:
done